lib/ladybug/energy_model/room.rb

```diff
@@ -138,6 +138,10 @@ def create_openstudio_object(openstudio_model)
           unless people.empty?
             people_object = people.get
             people_object.setSpace(openstudio_space)
+          else 
```

Comment:
@chriswmackey I added an else statement to create PeopleAbridged in case it doesn't exist.